IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/Cells/SSNewsCollectionCellVM.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SSNewsCollectionCellVM.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+
+final class SSNewsCollectionCellVM {
+    
+    let imageUrl: String
+    let date: Date
+    let description: String
+    let title: String
+    let author: String
+    let sourceURL: String
```

Comment:
I saw @bsztamfater-ms defined URLs as `URL` type instead of `string`, check with him so we take a consistent approach, and user one or the other.